### ISSUE TITLE
Avoid opening dirs as files

### ISSUE THREAD

**Issue by nfroidure** [1000ch/reachable-urls#2]
Doing this on my blog:
```sh
reachable-urls contents/**

⠋ Checking files(node:19427) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: EISDIR: illegal operation on a directory, read
⠸ Checking files
⠧ Checking files^C
```

Fails since your program try to open dirs as files. You should use an option to not list dirs whith your glob dependency. Something like that: https://github.com/nfroidure/metapak/blob/master/src/assets.js#L54

The above errro led to: https://github.com/1000ch/reachable-urls/issues/1

**Comment by 1000ch** [comment]
Definitely.

**Comment by 1000ch** [comment]
Fixed by 2ed48d1. Please update to `v0.9.1` 👍

**Comment by nfroidure** [comment]
Thanks
